### DISCUSSION TITLE
Open Most Recent File

### DISCUSSION THREAD

**Discussion by eisenbruch** [Victor-IX/Blender-Launcher-V2#58] [Ideas]
This feature in v2.0.24 changelog caught my eye:
"Added the possibility to launch a .blend file by dragging and dropping it onto a Blender version in the Library tab."

I think it would be great it there was also an option to launch the blender version you are currently hovering over, with the most recent file loaded automatically. 

This would save me nearly all my startup time, as I usually just open my last file. 
I don't want to find them in the file explorer, and then drag and drop it (even though that is a great feature and I am sure it will come in handy!)

**Comment by zeptofine** [comment]
IIRC Blender has an --open-last command so this should be done pretty easily!

**Reply by eisenbruch** [reply]
huzzah!

**Comment by eisenbruch** [comment]
when will this get released? or...how can I learn to compile myself and replace my installed version?

**Comment by eisenbruch** [comment]
i figured out how to build but when i try to run the exe i get this error:
Traceback (most recent call last):
  File "windows\main_window.py", line 75, in <module>
ModuleNotFoundError: No module named 'resources_rc'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 8, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "windows\main_window.py", line 85, in <module>
NameError: name 'exit' is not defined

help? :) trying to learn

**Reply by eisenbruch** [reply]
if I run pdm venv activate I get:
WARNING: Can't activate a non-venv Python C:/Users/spinn/AppData/Local/Programs/Python/Python310/python.EXE, you can
specify one with pdm venv activate <env_name>

not sure what i need to do

**Reply by zeptofine** [reply]
ah sht, the docs are wrong, I'll upload better ones
Try running `python build_style.py` and then `python source/main.py`

**Comment by eisenbruch** [comment]
ahhhh already loving this new feature :)

**Comment by zeptofine** [comment]
Yeah i really need to work on those docs. Tbh a lot of that stuff is automated for me due to how i setup my shell so it definitely needs to be worked on
